airbyte-integrations/connectors/source-slack/source_slack/source.py

```diff
@@ -322,11 +328,21 @@ def streams(self, config: Mapping[str, Any]) -> List[Stream]:
         default_start_date = pendulum.parse(config["start_date"])
         threads_lookback_window = pendulum.Duration(days=config["lookback_window"])
 
+        channel_filter = [s.strip() for s in config["channel_filter"].split(",") if s != ""]
```

Comment:
If you change the type of your configuration to `array` you'll receive a list value in this field and don't need this split logic to happen.

Reply:
done